sippy-ng/src/component_readiness/AdvancedOptions.js

```diff
@@ -123,6 +129,13 @@ export default function AdvancedOptions(props) {
               name="ignoreDisruption"
               color="primary"
             />
+            <p>Fallback Basis: {ignoreFallbackBasis ? 'ignore' : 'keep'}</p>
+            <Switch
+              checked={ignoreFallbackBasis}
+              onChange={handleChangeIgnoreFallbackBasis}
+              name="ignoreFallbackBasis"
+              color="primary"
+            />
```

Comment:
could it please have a tooltip to explain what it's for

Reply:
also i would find this less confusing if the boolean were reversed and so we had "Enable Fallback Basis" defaulting to false (for now)

Reply:
Agreed on the boolean reversal. I don't know if we can fit it but to me this is best described as Fallback To Best Past Release. Some version of that may help a little in the variable/param naming as well. (fallbackToBestPastRelease)

Reply:
I don't really disagree and likely wouldn't have coded it this way without prior art but what I did is consistent with what is [already there](https://github.com/openshift/sippy/blob/4fd58999afbe0caddd520af91f2b17d0dd029db2/sippy-ng/src/component_readiness/AdvancedOptions.js#L24) so I'm a little hesitant to refactor.

Reply:
With the move to include both the fallback basis as well as the requested basis I'm leaning towards removing this as an option and letting the default config (view?) determine if we use fallback or not.

Reply:
I see the [views](https://github.com/openshift/sippy/blob/master/config/views.yaml#L58)  are used to control the advanced options so I will investigate that vs. removal.

Reply:
Reversed the boolean and changed the label / parameter naming